### DISCUSSION TITLE
Organizzazione dei GH Projects

### DISCUSSION THREAD

**Discussion by ali-benny** [csunibo/csunibo.github.io#31] [Domande] [answered]
Sarebbe carino usare di più la sezione _Projects_ offerta da github per gestire meglio tutte le cose da fare in quanto anche segnalata nella wiki come un buon punto per vedere il "da farsi" e su cosa stiamo lavorando.
Attualmente sono presenti:
- "Altri corsi di Laurea"
- "Upload File" 

Aggiungerei qualcosa per gli strumenti che abbiamo come https://github.com/csunibo/informabot e https://github.com/csunibo/dynamik , e volendo anche altri (quali?)

**Comment by foxyseta** [comment]
Per me va bene. Mi raccomando ricordiamo che per una stessa repo possiamo anche fare più progetti se pensiamo che tematicamente ci siano più fili conduttori.

I progetti più facili da cui partire sono quelli piccolini tipo Course Description Merged, Raccoglitesi, ...

**Comment by ali-benny** [comment, marked as answer]
Forse sarebbe più utile creare un project per ogni team di sviluppo come frontend, backend, esperti-ci, bot ... no?

**Reply by ali-benny** [reply]
@foxyseta

**Reply by foxyseta** [reply]
Sì ci starebbe pure quello
